UnitySDK/Assets/ML-Agents/Examples/PushBlock/Scripts/PushBlockAcademy.cs

```diff
@@ -33,20 +25,4 @@ public class PushBlockAcademy : Academy
     /// material for a few seconds.
     /// </summary>
     public Material goalScoredMaterial;
-
```

Comment:
Why is this code removed ?

Reply:
are these part of the reset parameters?

Reply:
they weren't referenced by any code in the project